Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -172,22 +200,36 @@ - (id)init {
     return self;
 }
 
+@dynamic contactStore;
```

Comment:
I believe I need @dynamic to suppress auto-generation of the field backing the property, don't I?